TriblerGUI/widgets/trustpage/js/data_processor.js

```diff
@@ -188,24 +188,21 @@ function addMinMaxTransmission(response, interim) {
  *
  * @param {GraphResponseData} response - The response
  * @param {Object} interim - The interim result
- * @returns {{traffic_min: number, traffic_slope: number}}
+ * @returns {{min_total_traffic: number, max_total_traffic: number}}
  */
 function addTrafficFunction(response, interim) {
     var min = Number.MAX_VALUE,
-        max = 0;
+        max = Number.MIN_VALUE;
 
     interim.nodes.forEach(function (node) {
-       var sum = node.total_up + node.total_down;
-       if (sum < min) min = sum;
-       if (sum > max) max = sum;
+       var total_traffic = node.total_up + node.total_down;
+       min = Math.min(total_traffic, min);
+       max = Math.max(total_traffic, max);
     });
 
-    min = min === Number.MAX_VALUE ? 0 : Math.max(min, 0);
-    max = Math.max(max, 1);
-
     return {
-        traffic_min: min,
-        traffic_slope: 1 / (max - min)
+        min_total_traffic: min,
+        max_total_traffic: max
```

Comment:
This allows the min and max be negative in a bad case, but I don't know if it is possible for a node to have a negative amount of traffic.